faunadb/query.py

```diff
@@ -28,9 +28,9 @@ def var(var_name):
   return {"var": var_name}
 
 
-def if_expr(condition, true_expr, false_expr):
+def if_expr(condition, then, _else):
```

Comment:
The convention I've seen is to append the `_`, instead of prepending (as that's used for access modifiers)